aeron-archive/src/main/java/io/aeron/archive/ReplaySession.java

```diff
@@ -338,8 +353,12 @@ private int replay() throws IOException
                 result = publication.tryClaim(dataLength, bufferClaim);
                 if (result > 0)
                 {
+                    if (crcEnabled)
+                    {
+                        applyCrc(frameOffset, alignedLength);
```

Comment:
Apply before claim to keep window as small as possible to reduce cache misses and probability of buffer unblocking.

Reply:
Done